contracts/gas-service/AxelarGasService.sol

```diff
@@ -134,14 +138,20 @@ contract AxelarGasService is Upgradable, IAxelarGasService {
     }
 
     function refund(
-        address payable receiver,
+        address receiver,
         address token,
         uint256 amount
-    ) external onlyOwner {
+    ) external payable onlyOwner {
         if (receiver == address(0)) revert InvalidAddress();
 
         if (token == address(0)) {
-            receiver.transfer(amount);
+            if (amount <= address(this).balance) {
+                // solhint-disable-next-line avoid-low-level-calls
+                (bool sent, ) = receiver.call{ value: amount }('');
```

Comment:
Why is this better? `.transfer` only forwards 2300 gas as a stipend. Once using `.call`, the security implication is completely different as all gas is being forwarded.

Reply:
It is possible the `receiver` is a multisig contract that does something fancy on `receive` so 2300 gas is not enough. But you are right that the security implications are a lot more complicated but I can't think of any vulnerabilities (re-entrancy is covered by onlyOwner and memory is not used). If you can come up with something lmk, but even if we can't think of something i'm not sure if it's worh the risk just to support weird multisigs.

Reply:
@Foivos  One of those things people do is to check if there's any excessive gas provided, and if so the receiver contract would use as much as possible to get the gas token. In general, the 2300 stipend comes with a security reason that no SSTORE can be performed. I'd prefer not to provide any more gas other than that unless there's a use case already.